scripts/docker/gatkbase/Dockerfile

```diff
@@ -72,4 +72,6 @@ RUN mkdir $DOWNLOAD_DIR && \
     bash $DOWNLOAD_DIR/miniconda.sh -p $CONDA_PATH -b && \
     rm $DOWNLOAD_DIR/miniconda.sh && \
     conda clean -afy && \
+    conda config --set auto_update_conda false && \
+    conda config --set solver libmamba && \
```

Comment:
This is not strictly needed, since libmamba is the default solver in newer conda versions, but it might be worth hinting at the new use of libmamba here.